toolkit/scripts/toolchain/download_toolchain_rpm.sh

```diff
@@ -19,6 +19,8 @@ function usage() {
     echo "           until the RPM is successfully downloaded. Full URL will be <url>/<rpm_name>"
     echo "-c|--certificate: Optional path to a certificate file to use for the download"
     echo "-k|--private-key: Optional path to a private key file to use for the download"
+    echo "-e|--enforce-signatures: Optional flag to enforce RPM signatures"
+    echo "-g|--allowable-gpg-keys: Optional space separated list of GPG keys to allow for signature validation"
     exit 1
```

Comment:
Nit-pick and not even introduced in this change: I don't think `usage` should always `exit 1`. For instance, running the tool with `-h` shouldn't exit like an error.

Reply:
Sure, I can fix it. This script was added as a pre-cursor to this change.